packages/core/test/insured-bridge/OVM_L1BridgeRouter.js

```diff
@@ -0,0 +1,79 @@
+const hre = require("hardhat");
+const { didContractThrow, runDefaultFixture, ZERO_ADDRESS } = require("@uma/common");
+const { getContract, assertEventEmitted } = hre;
+
+const { assert } = require("chai");
+
+// Tested contract
+const BridgeRouter = getContract("OVM_L1BridgeRouter");
+const Finder = getContract("Finder");
+
+// Contract objects
+let bridgeRouter;
+let finder;
+
+describe("OVM_L1BridgeRouter", () => {
+  let accounts, owner, rando, l1CrossDomainMessenger;
+
+  before(async function () {
+    accounts = await web3.eth.getAccounts();
+    [owner, rando, l1CrossDomainMessenger] = accounts;
+    await runDefaultFixture(hre);
+    finder = await Finder.deployed();
+  });
+  beforeEach(async function () {
+    bridgeRouter = await BridgeRouter.new(finder.options.address, l1CrossDomainMessenger, owner, 7200).send({
+      from: owner,
+    });
+  });
+  describe("Admin functions", () => {
+    it("Ownership", async () => {
+      assert.equal(await bridgeRouter.methods.owner().call(), owner, "Owner not set on construction");
+
+      assert(
+        await didContractThrow(bridgeRouter.methods.transferOwnership(rando).send({ from: rando })),
+        "OnlyOwner modifier not enforced"
+      );
+
+      await bridgeRouter.methods.transferOwnership(rando).send({ from: owner });
+      assert.equal(await bridgeRouter.methods.owner().call(), rando, "Ownership not transferred");
+
+      await bridgeRouter.methods.renounceOwnership().send({ from: rando });
+      assert.equal(await bridgeRouter.methods.owner().call(), ZERO_ADDRESS, "Ownership not renounced");
+    });
+    it("Set Deposit contract", async () => {
+      const newDepositContract = rando;
+      assert(
+        await didContractThrow(bridgeRouter.methods.setDepositContract(newDepositContract).send({ from: rando })),
+        "OnlyOwner modifier not enforced"
+      );
+
+      const txn = await bridgeRouter.methods.setDepositContract(newDepositContract).send({ from: owner });
+      await assertEventEmitted(txn, bridgeRouter, "SetDepositContract", (ev) => {
+        return ev.l2DepositContract === newDepositContract;
+      });
+    });
+    it("Add whitelisted Token mapping", async () => {
```

Comment:
looks good. next will be adding in the cross-chain logic for this to call into optimism to whitelist on the other chain. can validate this with smockit :)